packages/web/src/instrumentations/console/index.ts

```diff
@@ -0,0 +1,28 @@
+import { agent, allLogLevels, LogLevel } from '@grafana/agent-core';
+import type { Instrumentation } from '@grafana/agent-core';
+
+export interface ConsoleInstrumentationOptions {
+  disabledLevels?: LogLevel[];
+}
+
+export class ConsoleInstrumentation implements Instrumentation {
```

Comment:
As a suggestion: instead of implementing `Instrumentation`, we can extend a `BasicInstrumentation` which defines the agent as a property. Doing so will allow us to skip an import in every instrumentation file while keeping an OOP-ish approach.

Reply:
-[ ] good idea